### PR TITLE
Feat: build order model

### PR DESCRIPTION
**Pull request by MahmoudRumaneh**

### CODE REVIEW

File: server/server/models/order.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const mongoose = requires('mongoose');
```
Comment (Dana-MO) [inline server/server/models/order.js:1]: Typo: requires should be require

File: server/server/models/order.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Item'
+  }],
+  total_price: {
+    type: Number,
+    required: true
+  },
+  status: {
+    type: String,
+    enum: ['in progress', 'completed'],
```
Comment (Dana-MO) [inline server/server/models/order.js:29]: Could we add an option canceled?

File: server/server/models/order.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  created_date: {
+    type: Date,
+    default: Date.now
+  },
+  updated_date: {
+    type: Date,
+    default: Date.now
+  },
+  note: String,
+  estimatedTime: Number //It will be like 50 Minutes
```
Comment (Dana-MO) [inline server/server/models/order.js:33]: Could we have estimatedTime be required?

**Review by Dana-MO — CHANGES_REQUESTED** [review]
Great Job, just some small adjustments

**Review by Hadeel-Ali-Obaid — APPROVED** [review]
Great job, I have nothing to add to what @Dana-MO mentioned.

**Review by Dana-MO — APPROVED** [review]

**Pull request merged**